guacamole/src/main/webapp/translations/en.json

```diff
@@ -536,6 +546,14 @@
         "FIELD_OPTION_FONT_SIZE_96"    : "96",
         "FIELD_OPTION_FONT_SIZE_EMPTY" : "",
 
+        "FIELD_OPTION_TERMINAL_TYPE_EMPTY" : "",
+        "FIELD_OPTION_TERMINAL_TYPE_XTERM" : "xterm",
+        "FIELD_OPTION_TERMINAL_TYPE_XTERM_256COLOR" : "xterm-256color",
+        "FIELD_OPTION_TERMINAL_TYPE_VT220" : "vt220",
+        "FIELD_OPTION_TERMINAL_TYPE_VT100" : "vt100",
+        "FIELD_OPTION_TERMINAL_TYPE_ANSI"  : "ansi",
+        "FIELD_OPTION_TERMINAL_TYPE_LINUX" : "linux",
```

Comment:
As mentioned above - for style consistency, please line up the ":" in this group of options.